apis/v1alpha2/policy_types.go

```diff
@@ -38,6 +38,16 @@ type PolicyTargetReference struct {
 	//
 	// +optional
 	Namespace *Namespace `json:"namespace,omitempty"`
+
+	// SectionName is the name of a section within the target resource. When
+	// unspecified, this targets the entire resource. In the following
+	// resources, SectionName is interpreted as the following:
+	// * Gateway: Listener Name
+	// * Route: Rule Name
```

Comment:
Route Rule Name doesnt exist in the spec yet, so should be added only after it has been formally added into the spec